price-feeder/cmd/price-feeder.go

```diff
@@ -12,12 +11,12 @@ import (
 	"syscall"
 	"time"
 
-	input "github.com/cosmos/cosmos-sdk/client/input"
 	"github.com/mitchellh/mapstructure"
 
 	"github.com/gorilla/mux"
 	"github.com/rs/zerolog"
 	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ssh/terminal"
```

Comment:
This package is deprecated:
> Error: SA1019: "golang.org/x/crypto/ssh/terminal" is deprecated: this package moved to golang.org/x/term.  (staticcheck)

Could you update and use `golang.org/x/term`?

Reply:
Completed in 9716bec44ccf733787e0d3e34dffb656b9135171